packages/serialization/multipart/src/multipartSerializationWriter.ts

```diff
@@ -34,47 +34,44 @@ export class MultipartSerializationWriter implements SerializationWriter {
 		if (key) {
 			this.writeRawStringValue(key);
 		}
-		if (value !== undefined) {
+		if (value) {
 			if (key) {
 				this.writeRawStringValue(": ");
 			}
 			this.writeRawStringValue(value);
 		}
-		this.writeRawStringValue("\r\n");
 	};
-	private writeRawStringValue = (value?: string | null): void => {
-		if (value !== undefined) {
-			const isNullValue = value === null;
-
-			this.writeByteArrayValue(undefined, new TextEncoder().encode(isNullValue ? "null" : value).buffer);
+	private writeRawStringValue = (value?: string): void => {
+		if (value) {
+			this.writeByteArrayValue(undefined, new TextEncoder().encode(value).buffer);
 		}
 	};
 	// eslint-disable-next-line @typescript-eslint/no-unused-vars
 	public writeBooleanValue = (key?: string, value?: boolean | null): void => {
 		throw new Error(`serialization of boolean values is not supported with multipart`);
 	};
 	// eslint-disable-next-line @typescript-eslint/no-unused-vars
-	public writeNumberValue = (key?: string, value?: number | null): void => {
+	public writeNumberValue = (key?: string, value?: number): void => {
```

Comment:
I believe all the `writeXXXXValue` methods should be reverted to include the `| null` in the parameter type. Otherwise they will no longer conform to the definitions defined in the `SerializationWriter` interface. 
https://github.com/microsoft/kiota-typescript/blob/434513968aa865908809a488f8b84200486a4d6f/packages/abstractions/src/serialization/serializationWriter.ts#L40